src/Network/Network/ApplicationGateway/FirewallConfiguration/AzureApplicationGatewayWebApplicationFirewallConfigurationBase.cs

```diff
@@ -43,7 +43,6 @@ public class AzureApplicationGatewayWebApplicationFirewallConfigurationBase : Ne
         [Parameter(
                Mandatory = false,
                HelpMessage = "The version of the rule set type.")]
-        [ValidateSet("3.0", "2.2.9")]
```

Comment:
Does it make sense to make this a PSArgumentCompleter, (which would provide tab completion but allow values outside the set) or are there no known values for this?

Reply:
No, because this list is dynamic. We don't want to keep track of PS changes. Also, there is a cmdlet to get all the versions.

Reply:
If there is a cmdlet to get all the versions, there should be a resource completer here that uses that API call.  This is one reason we have design reviews.